synapse/tests/test_lib_ast.py

```diff
@@ -492,6 +492,57 @@ async def test_ast_condsetoper(self):
                 q = '$foo=newp [test:str=foo :hehe*$foo=heval]'
                 nodes = await core.nodes(q)
 
+    async def test_ast_setmultioper(self):
+        async with self.getTestCore() as core:
+
+            nodes = await core.nodes('[ test:arrayprop="*" :ints=(1,) ]')
+            self.eq(nodes[0].get('ints'), (1,))
+
+            nodes = await core.nodes('test:arrayprop [ :ints++=([3, 4]) ]')
+            self.eq(nodes[0].get('ints'), (1, 3, 4))
+
+            nodes = await core.nodes('test:arrayprop [ :strs++=(foo, bar, baz) ]')
+            self.eq(nodes[0].get('strs'), ('foo', 'bar', 'baz'))
+
+            with self.raises(s_exc.BadTypeValu):
+                await core.nodes('test:arrayprop [ :ints++=(["newp", 5, 6]) ]')
+
+            nodes = await core.nodes('test:arrayprop [ :ints?++=(["newp", 5, 6]) ]')
+            self.eq(nodes[0].get('ints'), (1, 3, 4, 5, 6))
```

Comment:
I like that the try operator will still add the valid elements but it seems inconsistent with the other try edit operators (ival and maybe others?). For example, ```[ .seen ?= (20250212, newp) ]``` would fail to set both instead of only setting the start of the ival.

Reply:
Yeah that's still an invalid ival value though which fails to norm, so applying part of an invalid value seems weird. For something like a comp type, you wouldn't want to apply part of an invalid comp type when updating.

Reply:
I don't think it's inconsistent since an ival isn't really an array, it just a value that kinda looks like one.

Reply:
Makes sense, thanks for the explanation.